test/protosnewroute/hellomultiple.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
```

Comment:
Can you change the folder name `protosnewroute` to a more meaningful one, so that another person would understand your reasoning for the new folder?

Reply:
Perfect, Just changes protonewroute, to protomultiple